docs/post-response-codes.md

```diff
@@ -0,0 +1,80 @@
+# PostResponseCodes
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI (swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Async-V1-11
+
+## Output Message
+
+Synchronous POST operations must have responses with 200, 204 and default return codes. They also must not have other response codes.
+Long-running POST operations must have responses with 200, 202 and default return codes. They also must not have other response codes.
+
+## Description
+
+Synchronous POST must have 200 & 204 return codes.
+LRO POST must have 200 & 202 return codes with 'Location' header. LRO Operation must also add "x-ms-long-running-operation and x-ms-long-running-operation-options" to mark that it is a long running operation (in case of 202) and how it is tracked (Location header).
+
+## How to fix the violation
+
+For Synchronous POST operations add responses with 200, 204 and default return codes and make sure they don't have other response codes.
+
+For Long-running POST operations 
+1. Add responses with 200, 202 and default return codes and make sure they don't have other response codes 
```

Comment:
Change this line to reflect the comment I mentioned above

Reply:
use of a comma ',' doesnt really convey the proper meaning. Use an explicit 'and' instead

Reply:
pl check this everywhere